extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -84,7 +64,7 @@ public function index() {
 		$this->response->setVal( 'item_placeholder', wfMessage( 'wikiacuratedcontent-content-item' ) );
 		$this->response->setVal( 'name_placeholder', wfMessage( 'wikiacuratedcontent-content-name' ) );
 
-		$itemTemplate = $this->sendSelfRequest( self::ITEM_FUNCTION_NAME )->toString();
+		$itemTemplate = $this->sendSelfRequest( 'item' )->toString();
```

Comment:
Why? I don't hate, just ask ;) I thougt that we try to keep strings like this one in constants.